Makefile

```diff
@@ -0,0 +1,12 @@
+NAME = skilldlabs/frontend
+TAGS = d7 latest omega_4_3 omega_4_4 zen
+
+.PHONY: all build push
+
+all: build push
```

Comment:
Related to flow described here #16 
Would be great don't push images before them could be tested.
* make (build images localy)
* some script or testing section 
* push image if checks passed